### PR TITLE
Solve todo for question 6 in Matrix section.

### PR DESCRIPTION
**Pull request by JarneVdH**
Thanks for the repo, here is a small addition as follow-up of the LinkedIn post: https://www.linkedin.com/feed/update/urn:li:ugcPost:7130536207645585408?commentUrn=urn%3Ali%3Acomment%3A%28ugcPost%3A7130536207645585408%2C7130849621848797186%29&dashCommentUrn=urn%3Ali%3Afsd_comment%3A%287130849621848797186%2Curn%3Ali%3AugcPost%3A7130536207645585408%29

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\documentclass{article}
+ƒtodo\documentclass{article}
```
Comment (zafstojano) [inline main.tex]: I presume this is a typo when trying to search for the TODO? :)

File: main.tex [hunk 2]
```diff
@@ -242,7 +242,7 @@ \subsubsection{Matrices}
     \end{align*}
     Without explicitly using the equation for calculating determinants, what can we say about this matrix’s determinant? (Hint: rely on a property of this matrix to determine its determinant)
     \begin{answer}
-        \todo 
+        The third column equals (-2) $\times$ the first column, which means they are linearly dependent. As such, one of the eigenvalues will be 0. Since the determinant equals the factorisation of all eigenvalues, it will be 0 as well.
```
Comment (zafstojano) [inline main.tex:245]: lgtm!

**Review by zafstojano — CHANGES_REQUESTED** [review]

File: main.tex (hunk 1 above)
Comment (JarneVdH) [reply main.tex]: Wooops, you're right! 😅

Comment (JarneVdH) [inline main.tex]:
```suggestion
\documentclass{article}
```

**Review by zafstojano — APPROVED** [review]

**Pull request merged**